securedrop/tests/test_submission_cleanup.py

```diff
@@ -43,14 +43,19 @@ def test_delete_disconnected_fs_submissions(journalist_app, config):
     """
     Test that files in the store without corresponding Submission records are deleted.
     """
-    source, codename = utils.db_helper.init_source()
+    source, _ = utils.db_helper.init_source()
 
     # make two submissions
     utils.db_helper.submit(source, 2)
     source_filesystem_id = source.filesystem_id
     submission_filename = source.submissions[0].filename
     disconnect_path = os.path.join(config.STORE_DIR, source_filesystem_id, submission_filename)
 
+    # make two replies, to make sure that their files are not seen
+    # as disconnects
+    journalist, _ = utils.db_helper.init_journalist("Mary", "Lane")
```

Comment:
Would it make sense to also delete one of the reply records here, to verify that reply files are being deleted correctly?

Reply:
Reply deletion isn't part of submission cleanup (but see below), and it is already tested in  `test_integration.py` and `test_journalist.py`. It is worth nothing that [`test_integration._helper_test_reply`](https://github.com/freedomofpress/securedrop/blob/ae0ff0c36b80062097826852b02f667243c45a16/securedrop/tests/test_integration.py#L350) does not check that the file is actually gone, as is done in [`test_submit_file`](https://github.com/freedomofpress/securedrop/blob/ae0ff0c36b80062097826852b02f667243c45a16/securedrop/tests/test_integration.py#L242), but that _is_ tested, albeit somewhat indirectly, in [`test_journalist.test_delete_source_deletes_docs_on_disk`](https://github.com/freedomofpress/securedrop/blob/ae0ff0c36b80062097826852b02f667243c45a16/securedrop/tests/test_journalist.py#L1683), where a source collection with submissions and replies is deleted.

So ... it's probably worth reviewing all the tests involving reply deletion, in a separate issue. And I think if we have submission cleanup tools, we should have them for replies as well. Replies are smaller and less likely to have their deletion process interrupted as has happened for large submissions, but it is still technically possible for their files and database records to be disconnected. But I think that's out of scope for this fix, and possibly for 1.0.0.

Reply:
agreed, let's keep this scoped to submissions for 1.0.0